akka-http-core/src/main/scala/akka/http/impl/engine/ws/FrameOutHandler.scala

```diff
@@ -44,6 +43,9 @@ private[http] class FrameOutHandler(serverSide: Boolean, _closeTimeout: FiniteDu
       case UserHandlerCompleted ⇒
         become(new WaitingForPeerCloseFrame())
         ctx.push(FrameEvent.closeFrame(Protocol.CloseCodes.Regular))
+      case UserHandlerErredOut(ex) ⇒
+        become(new WaitingForPeerCloseFrame())
+        ctx.push(FrameEvent.closeFrame(Protocol.CloseCodes.UnexpectedCondition, "internal error"))
```

Comment:
I cannot say for sure right now. I will be able to look into it next week. AFAICS either this case didn't happen (in which case I should have documented it, or otherwise, it is indeed missing (and missing a test as well). So, could you add a ticket to figure this out? I will then have a look next week. Thanks!